lib/openstudio/aws/aws.rb

```diff
@@ -141,22 +143,30 @@ def create_server(options = {}, instances_filename = 'server_data.json')
 
         # save the worker pem and public to the directory
         # presently, this will always overwrite the worker key, is that okay? Is this really needed later?
-        @os_aws.save_worker_keys('.')
+        @os_aws.save_worker_keys(@save_directory)
 
         # if instance_data[:ebs_volume_id]
         #   server_options[:ebs_volume_id] = instance_data[:ebs_volume_id]
         # end
 
         @os_aws.launch_server(options[:image_id], options[:instance_type], server_options)
+      end
 
-        @instances_file_name = instances_filename
-        save_cluster_json @instances_file_name
-
+      # Write out to the terminal the connection information for the servers and workers
+      #
+      # @return [nil] Only prints to the screen. No return is expected
+      def print_connection_info
```

Comment:
Assignment Branch Condition size for print_connection_info is too high. [18.03/15]